web/packages/teleterm/package.json

```diff
@@ -54,7 +54,7 @@
     "react-dnd": "^14.0.4",
     "react-dnd-html5-backend": "^14.0.2",
     "whatwg-url": "^14.2.0",
-    "zod": "^3.25.67",
+    "zod": "^4.0.17",
```

Comment:
> For example, zod v4 appears to produce less informative errors

With a change like this, I'd expect people to complain about it in the issue tracker.